lib/Target.js

```diff
@@ -38,7 +38,7 @@ class Target {
    * @return {!Promise<?Page>}
    */
   async page() {
-    if (this._targetInfo.type === 'page' && !this._pagePromise) {
+    if ((this._targetInfo.type === 'page' || this._targetInfo.type === 'background_page') && !this._pagePromise) {
```

Comment:
can you please update `target.page()` documentation and mention that it now works with `'background_page'` type?

Also: can you please add the `'background_page'` type to  `target.type()` documentation?

Reply:
added to `target.page()`. Regarding `target.type()` - it was already there :)